src/test/java/hec/ensemble/ReadWriteEnsembleTest.java

```diff
@@ -0,0 +1,45 @@
+package hec.ensemble;
+
+import hec.JdbcTimeSeriesDatabase;
+import hec.TimeSeriesDatabase;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadWriteEnsembleTest {
+    @Test
+    public void ReadWrite() throws Exception {
```

Comment:
Please add timing and track all changes with timings so we can determine when we improve or decrement performance